### ISSUE TITLE
Can't seem to figure out how to add authors

### ISSUE THREAD

**Issue by michael** [substance/texture#893]
Reported by Giuliano. 

They can be added using the "Add -> Author" menu from the toolbar.

We can add additional links to the metadata body to make this clearer. But needs a bit of thought first.

**Comment by gmaciocci** [comment]
Where's the Add menu?

![screen shot 2018-09-19 at 14 56 11](https://user-images.githubusercontent.com/12699608/45757816-2aebc200-bc1c-11e8-9b83-89ec49e2ff1f.png)

![screen shot 2018-09-19 at 14 55 35](https://user-images.githubusercontent.com/12699608/45757781-15769800-bc1c-11e8-8abf-a1bc1a9d7146.png)

**Comment by michael** [comment]
In the metadata view.

![image](https://user-images.githubusercontent.com/2931/45758954-1e209b80-bc27-11e8-83b1-97233c885e40.png)

**Comment by michael** [comment]
That's the bad side of introducing modes. You loose orientation. ;)

**Comment by michael** [comment]
One silly idea to get rid of the mode switch: What about rendering the metadata section right below the manuscript with a clear separator.

**Comment by lpanebr** [comment]
> One silly idea to get rid of the mode switch: What about rendering the metadata section right below the manuscript with a clear separator.

I don't think this is silly at all. A better approach I think would be to spread some of them (authors, affiliations, author notes etc) in the front and others in the back (references, article notes and footnotes etc) something like I've just described in #894
